src/main/pages/overview/single-multi-user.adoc

```diff
@@ -8,26 +8,39 @@ folder: overview
 
 ---
 
-Che is shipped as two different flavors - single and multi user. A single user Che *has no components that provide multi tenancy and permissions*. Thus, Che server and workspaces are not secured. This makes a single user Che a good choice for developers working locally.
+Eclipse Che is available in two different modes: single-user and multi-user.
 
-A multi user Che provides multi-tenancy i.e. *users accounts and workspaces are isolated and secured* with Keycloak tokens. Che uses http://www.keycloak.org/[Keycloak] as a mechanism to register, manage and authenticate users. Permissions API regulates access to different entities in Che, such as workspaces, stacks, recipes, organizations etc. User information is stored in a persistent DB that supports migrations (PostgreSQL).
+**Single-user Che**
 
-If you plan using Che just on your local machine or just evaluate the platform, it is reasonable to start with a *single-user Che*.
+If you plan to use Che on your local machine or want to evaluate the platform, start with single-user Che. The advantages for using single-user Che are:
 
-*Single User Che Pros:*
+* The command line interface pulls fewer images
+* The user dashboard is quickly available (no login)
 
-* The CLI will pull fewer Images
-* You will get to the User Dashboard more quickly (no login)
+Single-user Che does not provide multi-tenancy nor permissions for entities such as workspaces. Che server and workspaces are not secure with single-user Che. 
 
-*Multi-User Che Pros*
+The following links have more information on installation:
 
-* A Fully functional web IDE with fine grained access controls
-* A standalone Keycloak server that supports users federation and identity providers
+* To install Che in the single-user mode on Docker, see link:docker-single-user[Single-user on Docker] 
 
-By default *Che gets deployed as a single user* assembly both on Docker and OpenShift. Special flags must be provided to enable multi-user functionality.
+* To install Che in the single-user mode on OpenShift, see link:openshift-single-user[Single-user on OpenShift].
+
+**Multi-user Che**
+
+Multi-user Che provides multi-tenancy. Che uses http://www.keycloak.org[Keycloak] as a mechanism to register, manage, and authenticate users. Keycloak tokens provide security for user accounts and workspaces. The Permissions API regulates access to different entities such as workspaces, stacks, recipes, and organizations. User information is stored in a persistent PostgreSQL database which supports migrations.
+
+The advantages for using multi-user Che are:
+
+* The fully functional web IDE supports precise access controls
+* The standalone Keycloak server supports users federation and identity providers
+
+By default Che deploys in the single-user mode for Docker and OpenShift. You can use special flags to enable multi-user functionality.
```

Comment:
* -> _By default, Che_
* -> _-the- mode_
* in this case, passive mode is required because it's not Che that deploys something (or itself); instead, Che is deployed (by the user, admin, Ansible playbook, etc.)